src/Controller/SentencesListsController.php

```diff
@@ -169,13 +175,16 @@ public function show($id = null, $translationsLang = null)
         $this->loadModel('Sentences');
         $this->loadModel('SentencesSentencesLists');
 
-        $query = $this->SentencesSentencesLists->find()
-            ->where(['sentences_list_id' => $id])
+        $query = $this->SentencesSentencesLists->find();
+        if ($lang!="und") {
+            $query->where(['lang IS' => $lang == 'unknown' ? null : $lang]);
```

Comment:
I think you can remove the `IS` keyword now.

Reply:
the docs for 3.8 seem to indicate it is necessary for automatic `IS NULL` creation.

https://book.cakephp.org/3/en/orm/query-builder.html#automatic-is-null-creation.